packages/website/components/Sidebar.tsx

```diff
@@ -143,6 +144,16 @@ export function Sidebar({ currentPage = '/' }: Props) {
 
       {activeSection === WEBSITE_SECTION.COMPONENTS && (
         <>
+          <SidebarSection
+            links={[
+              {
+                title: 'Proposal in GitHub',
```

Comment:
@gui-santos dunno, maybe something like this? or better. "Idea in GitHub" ? WDYT?

Reply:
looks good to me, I'm not the best with copy

I tried this as well

![Screenshot 2022-01-26 at 11 14 14](https://user-images.githubusercontent.com/6597467/151155872-3dff5bff-fdd8-403a-94d0-4343c314b602.png)

But we can always change copy later, I just told you because the scrollbar was braking the layout